### DISCUSSION TITLE
*** Fruits Party *** 알파 버전

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#689] [4. General]
안녕하세요 ~ 

Fruits Party 알파 버전을 공유 해요 ~ 

이번 버전에서는 게임이 한싸이클 돌아가게끔 만드는데 목표를 두었어요 

게임에 참여할 플레이어를 기다린후 함께 입장해요 

게임이 시작 되면 임의 위치에서 생성되어 플레이 할수 있게 했어요 

과일 버튼을 누르면 자동으로 가장 가까운 근처 목표를 찾아 공격 할 수 있게 했습니다 

더미 오브젝트뒤에 숨는다면 다른 플레이어의 공격을 피할수도 있어요 

선인장같은 주변 오브젝트에 숨을수도 있게 만들어 보았어요 

은폐 엄폐를 전략적으로 이용했으면 하는 바램이였습니다 ~ 

게임의 종료는 승리 패배 인데요 

그동안 플레이 했던 전적 (승/패)를 기록해 보여주고 승률도 함께 보여주도록 만들어 봤습니다 

롤이나 오버워치 같은 게임에서 영감을 얻어 점수에 따른 티어도 보여줄수 있도록 만들었어요 

아직 버그도 많고 컨텐츠도 많이 부족 하지만 6월 중순이나 말쯤에 서비스 예정이기에 열심히 만들어 나가고 있답니다 

종종 개발 진행을 공유해드릴게요 ~ 그럼 빠이요 ~ 

https://user-images.githubusercontent.com/50188795/166185695-ae3f7663-fa81-44b9-9aab-0aacb05751a4.mp4

**매번 질문에 답변을 주신 모든 분들 감사합니다 ~ 👍** 

가끔 유튭 생방 켜놓고 개발도 하니 놀러오세요 ~ 

https://youtube.com/playlist?list=PLTFRwWXfOIYCOdgFbutBxPBbvLkb_2GL9

**Comment by zepetomono** [comment]
와 감사합니다!

**Comment by x4cf** [comment]
멋져요!
